react/components/SKUSelector/index.tsx

```diff
@@ -291,6 +291,8 @@ const SKUSelectorContainer: FC<Props> = ({
 
   const dispatch = useProductDispatch()
 
+  const getSkuId = () => query?.skuId
```

Comment:
I think this should be a dependency of the `onSelectedItem` `useCallback`. I think you'll need to wrap it on a `useCallback` of it's own